espresso.php

```diff
@@ -61,7 +61,7 @@ function espresso_duplicate_plugin_error()
     }
     add_action('admin_notices', 'espresso_duplicate_plugin_error', 1);
 } else {
-    define('EE_MIN_PHP_VER_REQUIRED', '5.4.0');
+    define('EE_MIN_PHP_VER_REQUIRED', '5.6.2');
```

Comment:
👍

Reply:
oh ya, forgot I meant to post a link showing that that is WP's minimum now:
https://wordpress.org/support/article/requirements/